addons/mail/models/res_users.py

```diff
@@ -137,6 +137,28 @@ def _get_portal_access_update_body(self, access_granted):
             return '%s (%s)' % (body, self.partner_id.email)
         return body
 
+    @api.model
```

Comment:
Not a model anymore ^^ moreover we should probably clearly separate current user from deactivated users in messages :) we might call this method to deactivate some portal users in batch (as an erp manager for example)